apps/svelte.dev/content/blog/2022-12-14-announcing-sveltekit-1.0.md

```diff
@@ -88,7 +88,7 @@ Back in early 2021, when we adopted Vite, we were the first major application fr
 
 We’ve had terrific support from teams at [Vercel](https://vercel.com), [Netlify](https://netlify.app/) and [Cloudflare](https://www.cloudflare.com/) to make zero-config deployments for those platforms possible.
 
-Our friends at [StackBlitz](https://stackblitz.com/) worked feverishly to make [learn.svelte.dev](https://learn.svelte.dev), our first-of-its-kind [WebContainer](https://blog.stackblitz.com/posts/introducing-webcontainers/)-powered interactive tutorial, a reality.
+Our friends at [StackBlitz](https://stackblitz.com/) worked feverishly to make [our new tutorial](/tutorial), the first-of-its-kind [WebContainer](https://blog.stackblitz.com/posts/introducing-webcontainers/)-powered interactive tutorial, a reality.
```

Comment:
let's at least revert this, because this _is_ messing with history in a way that feels wrong. like, `learn.svelte.dev` appears in talks and blog posts

```suggestion
Our friends at [StackBlitz](https://stackblitz.com/) worked feverishly to make [learn.svelte.dev](https://learn.svelte.dev), our first-of-its-kind [WebContainer](https://blog.stackblitz.com/posts/introducing-webcontainers/)-powered interactive tutorial, a reality.
```

Reply:
(and if we're reverting that, we should probably revert the other mention also in this document)

Reply:
done. I'm fine reverting all of them honestly except for the last two which would be otherwise broken